common.blocks/issue/issue.bemtree.js

```diff
@@ -40,15 +40,15 @@ block('issue').content()(function() {
             elem: 'content',
             content: issue.html
         },
-        this.data.url.pathname === '/' ? {
+        this.data.url.pathname.match(/^\/\d+\/$/g) !== null ? {
```

Comment:
I prefer not to use regexps as long as it possible. So I'd better use something like:
```js
this.data.url.pathname === '/' || this.data.view === '404'
```